topo_processor/cog/create_cog_test.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+import pytest
+
+from topo_processor.cog.command import Command
+from topo_processor.cog.create_cog import create_cog
+
+
+@pytest.fixture(autouse=True)
+def delete_cog():
+    """Automatically runs before and after each test"""
+    yield
+    output_path = os.path.join(os.getcwd(), "CROWN_399_E_49.tiff.LZW.cog.tiff")
+    if os.path.isfile(output_path):
+        os.remove(output_path)
+
+
+@pytest.mark.asyncio
+async def test_create_cog():
+    input_file = os.path.join(os.getcwd(), "test_data", "tiffs", "CROWN_399_E_49.tiff")
+    output_dir = os.getcwd()
+    compression_method = "LZW"
+    await create_cog(input_file, output_dir, compression_method)
+
+
+@pytest.mark.asyncio
+async def test_command_local():
+    cmd = Command("ls")
+    cmd.append_arg("-l")
+    await cmd.run()
```

Comment:
What is this actually testing? need some sort of assertion about what is returned, exit code, outputs etc..